lib/puma/runner.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'puma/server'
 require 'puma/const'
+require 'puma/minissl/context_builder'
```

Comment:
Nice call. I'm not gonna review that commit because it looks like a straight copy-paste into the new class.